<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>211</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineAddDataset"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="btnAddDataset">
     <property name="text">
      <string>Add dataset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QComboBox" name="typeSelector">
     <item>
      <property name="text">
       <string>Widget::Bar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Widget::Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Widget::Pie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Widget::Polar</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QFrame" name="chartFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Select type</string>
     </property>
     <property name="buddy">
      <cstring>typeSelector</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="leadingLabel">
     <property name="text">
      <string>Leading</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>181</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="leadingSelector"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
